File: w01/d04/Nathaniel_Tuvera/calculator/calculator.rb
class Calculator
  def add(x, y)
    x + y
  end

  def subtract(x, y)
    x - y
  end

  def divide (x, y)
    x.to_f / y.to_f
  end

  def multiply (x, y)
    x * y
  end

  def div_zero (x)
    if x / 0
      return NaN
  end
end

File: w07/d05/Gardner_Lonsberry/hw/jasmine_test/src/calculator.js
function calculator() {
}

calculator.prototype = {
  add: function(x, y){
    return x + y;
  },

  subtract: function(x, y){
    return x - y;
  },

  multiply: function(x, y){
    return x * y;
  }
}

var calculator = new calculator